migration/migration.go

```diff
@@ -202,6 +205,70 @@ func getCurrentVersion(db *sql.Tx) (int64, error) {
 	return current, nil
 }
 
+// BootstrapWorkItemLinking makes sure the database is populated with the correct work item link stuff (e.g. category and some basic types)
+func BootstrapWorkItemLinking(ctx context.Context, db *gorm.DB, linkCatRepo *models.GormWorkItemLinkCategoryRepository, linkTypeRepo *models.GormWorkItemLinkTypeRepository) error {
+	if err := createOrUpdateWorkItemLinkCategory(models.SystemWorkItemLinkCategorySystem, "The system category is reserved for link types that are to be manipulated by the system only.", ctx, linkCatRepo, db); err != nil {
+		return err
+	}
+	if err := createOrUpdateWorkItemLinkCategory(models.SystemWorkItemLinkCategoryUser, "The user category is reserved for link types that can to be manipulated by the user.", ctx, linkCatRepo, db); err != nil {
+		return err
+	}
+	if err := createOrUpdateWorkItemLinkType(models.SystemWorkItemLinkTypeBugBlocker, "one bug blocks another", models.TopologyNetwork, "blocks", "blocked by", models.SystemBug, models.SystemBug, models.SystemWorkItemLinkCategorySystem, ctx, linkCatRepo, linkTypeRepo, db); err != nil {
+		return err
+	}
+	return nil
+}
+
+func createOrUpdateWorkItemLinkCategory(name string, description string, ctx context.Context, linkCatRepo *models.GormWorkItemLinkCategoryRepository, db *gorm.DB) error {
+	cat, err := linkCatRepo.LoadCategoryFromDB(ctx, name)
```

Comment:
The repository has a "save" method. Why not use that instead of going directly to the db?